app/src/main/java/com/github/se/cyrcle/ui/profile/CreateProfileScreen.kt

```diff
@@ -1,55 +1,57 @@
 import com.github.se.cyrcle.ui.theme.atoms.Text
-import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
+import com.github.se.cyrcle.ui.theme.molecules.BottomNavigationBar
 
 @Composable
 fun CreateProfileScreen(navigationActions: NavigationActions, userViewModel: UserViewModel) {
   val screenTitle = stringResource(R.string.profile_screen_title)
 
   Scaffold(
       modifier = Modifier.fillMaxSize().testTag("CreateProfileScreen"),
-      topBar = { TopAppBar(navigationActions, screenTitle) }) { padding ->
-        Column(
-            modifier = Modifier.fillMaxSize().padding(padding),
-            verticalArrangement = Arrangement.Center) {
-              Text("TODO")
-
-              // TODO Toast text need to take inspiration from this
-              // val failSignInMsg = stringResource(R.string.sign_in_failed_toast)
-              // val successSignInMsg = stringResource(R.string.sign_in_successful_toast)
-              //
-              // GoogleSignInButton(
-              //    onAuthComplete = { result: AuthResult ->
-              //      val user =
-              //          User(
-              //              userId = result.user?.uid ?: "",
-              //              username = result.user?.displayName ?: "",
-              //              email = result.user?.email ?: "")
-              //      Log.d("Cyrcle", "User account created: ${result.user?.displayName}")
-              //      Toast.makeText(context, "Welcome ${user.username}!", Toast.LENGTH_LONG).show()
-
-              //      userViewModel.addUser(user)
-              //      userViewModel.setCurrentUser(user)
-              //      navigationActions.navigateTo(TopLevelDestinations.MAP)
-              //    },
-              //    onAuthError = { e: ApiException ->
-              //      Log.e("Cyrcle", "Failed to sign in: ${e.statusCode}")
-              //      Toast.makeText(context, "Account creation failed...",
-              // Toast.LENGTH_LONG).show()
-              //    })
-            }
+      bottomBar = { BottomNavigationBar(navigationActions, selectedItem = Route.PROFILE) },
+  ) { padding ->
+    Box(modifier = Modifier.padding(padding)) {
+      Column(
+          modifier = Modifier.fillMaxSize().padding(16.dp),
+          verticalArrangement = Arrangement.Center,
+          horizontalAlignment = Alignment.CenterHorizontally,
+      ) {
+        Text(
+            "You need to be signed-in to modify your profile",
+            style = MaterialTheme.typography.titleLarge)
+        Spacer(modifier = Modifier.height(16.dp))
+        Text(
+            "Sign in now to add new spots, share reviews, and help fellow cyclists find the perfect parking space.")
+        Spacer(modifier = Modifier.height(8.dp))
+        Button(
+            text = "Go back to sign in",
+            onClick = {
+              // navigate to the route to not clear backstack
+              navigationActions.navigateTo(Route.AUTH)
```

Comment:
You probably wanted to use `navigationAction.navigateTo(TopLevelDestinations.AUTH)` since the go back button from AUTH after clicking here does not make us quit the app

Reply:
Here I specifically don't use the TLD because it clears the back stack and so the user can't use the back action if he changes his mind and doesn't want to login anymore.

Reply:
Well a solution could be to do as matteo said and maybe add an `AlertDialog` composable that ask the user if he is certain about going back to the login screen.

Reply:
I could see why it's a valid approach, it just changes the behaviour of the screen in both scenarios but it's ok I guess.

Reply:
There is no need, this works perfectly right now : 
1. The guest user decides to sign in
2. He is redirected on the auth screen that is put on top of the stack instead of clearing it
3. This means now the user can back track, change his mind, use the go back and resume his session on the profile screen.
3.bis He logs in, is redirected to the map screen, the back stack is cleared. 

Am I missing something ? do you want the back stack to be cleared ? does not seem very conviniant to me

Reply:
No no it makes sense, the fact that the stack is cleared when navigating to the map solves my "grudge". And I still had a mixup between TLDs and screens but its good now